Source/WebCore/page/scrolling/ScrollAnchoringController.h

```diff
@@ -41,12 +41,18 @@ class ScrollAnchoringController final {
     void invalidateAnchorElement();
     void updateScrollPosition();
     void selectAnchorElement();
+    void chooseAnchorElement();
+    void updateAnchorElement();
+    void printAnchorElement();
     LocalFrameView& frameView() { return m_frameView; }
 
 private:
     LocalFrameView& m_frameView;
+    FloatSize computeOffset(RenderBox& candidate);
     WeakPtr<Element, WeakPtrImplWithEventTargetData> m_anchorElement;
-    FloatPoint m_lastPositionForAnchorElement;
+    FloatSize m_lastOffsetForAnchorElement;
+    bool m_midAdjustingScrollPositionForAnchorElement { false };
+    bool m_isQueued;
```

Comment:
initialize. It's not obvious what "queued" means.

Reply:
Changed to m_isQueuedForScrollPositionUpdate. This indicates that this controller has a pending scroll position update and is on the set held by the FrameView.